test/GoodGhostingGasEstimate.test.js

```diff
@@ -200,11 +207,25 @@ contract("GoodGhostingGasEstimate", (accounts) => {
             } else {
                 const expectedAmount = new BN(await goodGhosting.adminFeeAmount.call({from: admin}));
                 const result = await goodGhosting.adminFeeWithdraw({ from: admin });
-                truffleAssert.eventEmitted(
-                    result,
-                    "AdminWithdrawal",
-                    (ev) => expectedAmount.eq(ev.adminFeeAmount)
-                );
+                if (GoodGhostingArtifact === GoodGhostingPolygon) {
+                    rewardToken = new web3.eth.Contract(daiABI, providersConfigs.wmatic);
+                    const adminMaticBalance = new BN(await rewardToken.methods.balanceOf(admin).call({ from: admin }));
```

Comment:
based on current implementation, admin doesn't take a cut from the matic rewards, except when there's no winner.
this test fails to catch this because it does't compare the admin matic balance before and after adminFeeWithdrawal.

Reply:
yeah thanks for pointing this out just slipped from my head to check this